lib/from_openstudio/program_type.rb

```diff
@@ -0,0 +1,125 @@
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# (1) Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# (2) Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'honeybee/program_type'
+require 'to_openstudio/model_object'
+
+module Honeybee
+    class ProgramTypeAbridged < ModelObject
+
+        def self.from_programtype(programtype)
+            # create an empty hash
+            hash = {}
+            hash[:type] = 'ProgramTypeAbridged'
+            # set hash values from OpenStudio Object
+            hash[:identifier] = clean_name(programtype.nameString)
+            unless programtype.displayName.empty?
+                hash[:display_name] = (programtype.displayName.get).force_encoding("UTF-8")
+            end
+            unless programtype.people.empty?
+                programtype.people.each do |people|
+                    people_def = people.peopleDefinition
+                    # Only translate if people per floor area is specified
+                    # Check if schedule exists and is of the correct type
+                    if !people_def.peopleperSpaceFloorArea.empty? && !people.numberofPeopleSchedule.empty?
+                        sch = people.numberofPeopleSchedule.get.scheduleTypeLimits.get.unitType
+                        # TODO Fractional isn't listed in schedule type limits unit type
+                        if sch == 'Dimensionless'
```

Comment:
@chriswmackey as per the schema these should be `Fractional` schedules but we havent added a `Fractional` unit type to the enum for schedule type limits